benchmarks-next/Dockerfile.default

```diff
@@ -0,0 +1,7 @@
+FROM emscripten/emsdk:2.0.8
```

Comment:
Maybe name this file `EmscriptenDockerfile.default` so we can eventually have similar deduplication for rust containers?

Reply:
I went with `Dockerfile.emscripten` since I've seen/used that name format before.